### PR TITLE
create palindrome.py

### PR DESCRIPTION
**Pull request by skkuYJ**
(Put an `X` inside the `[ ]` to denote check mark `[X]`.)

- [X] **If creating a new file :**
  - [X] added links to it in the README files ?
  - [X] included tests with it ?
  - [X] added description (overview of algorithm, time and space complexity, and possible edge case) in docstrings ?

- [X] **if done some changes :**
  - [X] wrote short description in the PR explaining what the changes do ?
  - [ ] Fixes #[issue number] if related to any issue

- [ ] **other**

### CODE REVIEW

File: algorithms/dp/palindrome.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+N1=[1,2,1,3,1,2,1]
+M1=[[1,3],[2,5],[3,3],[5,7]]
+
+N2=[1, 2, 3, 1, 5, 1, 3, 3, 1]
+M2=[[1,3],[2,5],[4,6],[6,9],[3,7]]
+'''
+
+def palindrome(list_n,list_m) :
+        n=len(list_n)
```
Comment (keon) [inline algorithms/dp/palindrome.py]: plz use 4 spaces

Comment (skkuYJ) [reply algorithms/dp/palindrome.py]: what do you mean by 4 spaces?

Comment (skkuYJ) [reply algorithms/dp/palindrome.py]: oh i fixed it. please review this.

File: algorithms/dp/palindrome.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+M1=[[1,3],[2,5],[3,3],[5,7]]
+
+N2=[1, 2, 3, 1, 5, 1, 3, 3, 1]
+M2=[[1,3],[2,5],[4,6],[6,9],[3,7]]
+'''
+
+def palindrome(list_n,list_m) :
+    n=len(list_n)
+    output=[]
+    d=[[0 for i in range(n)] for j in range(n)]
```
Comment (keon) [inline algorithms/dp/palindrome.py]: please use more specific variable names
`n`, `d`, `m`, `s`, `e`

**Comment by keon** [comment]
Closing as stale — the repo was completely restructured in v1.0.0. Thank you for the contribution\!
